alicloud/resource_alicloud_drds_instance_test.go

```diff
@@ -0,0 +1,121 @@
+package alicloud
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+
+	"github.com/aliyun/alibaba-cloud-sdk-go/services/drds"
+	"github.com/hashicorp/terraform/terraform"
+	"github.com/terraform-providers/terraform-provider-alicloud/alicloud/connectivity"
+)
+
+func TestAccAlicloudDRDSInstance_Basic(t *testing.T) {
+	var instance drds.DescribeDrdsInstanceResponse
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
+		},
+		IDRefreshName: "alicloud_drds_instance.basic",
+		Providers:     testAccProviders,
+		CheckDestroy:  testAccCheckDRDSInstanceDestroy,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccDrdsInstance,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckDRDSInstanceExist(
+						"alicloud_drds_instance.basic", &instance),
+				),
```

Comment:
Please check all of attributes.

Reply:
fix

Reply:
I think there is still missing several attributes.